### ISSUE TITLE
Front page news stories - layout

### ISSUE THREAD

**Issue by GilesGibson** [folklabs/hernehill.org.uk#480]
hernehill.org site, post content transfer.

Looking at how the front page is after the old content has been ported across would it look better if we had more lines of each news story listed before the "more" icon? Maybe at least a paragraph or so? Doesn't leave much to tempt people to click on to the main story.

![image](https://cloud.githubusercontent.com/assets/6060985/6212454/5d802f10-b5dd-11e4-9650-403c95b5846f.png)

**Comment by djwesto** [comment]
No, the reason we've chosen that amount of main paragraph text to display under the article headline is that all front page featured stories should have an image. If they don't have an image, then they do look a bit empty. However, if we were to allow more text to be displayed, then when an image is sitting next to the story, a huge white gap will appear underneath the image.

I vote we keep this layout for V1.0 and review after several months' worth of content has been added so we can see how many of your authors are actually adding images to the articles you're featuring.

**Comment by GilesGibson** [comment]
Is it possible to change the layout dependent on there being an image? if no image then more text, if image then reduced text?

The layout issue was raised by Yan as he felt there should be more text to tease you in to the full story.

**Comment by pmackay** [comment]
We've discussed that, there may be ways to do it, but its not a
straightforward thing to work out, so would favour investigating later.

**Comment by GilesGibson** [comment]
OK, leave on backburner. See what feedback we get from the other testers.

**Comment by pmackay** [comment]
Another aspect to this is that the layout is more challenging because we're requiring an image to the left of text. What about having the image above? Compare and contrast on http://www.brixtonbuzz.com/ - they use both layout types on homepage.
